resources/Import-Grammars-Test/nodeTypesChanges_Reference.csv

```diff
@@ -566,16 +566,16 @@ Name:;Added to language(s):;Removed from language(s):;Remains in language(s):;Us
 "super";;kt;java,js,ts;;
```

Comment:
I think it is not ideal to have a file like this which needs to be maintained. This information is available in other files already (nodeTypesConfig), isn't it?

Reply:
I left this when updating the import script to later add an automated test case for it. Additionally to the old nodeTypesConfig.json, we would also need fixed copies of the node-types.json of the grammars to get a reproducable result.

No, it is not the same as even our current nodeTypesConfig, as it does not include any of the manually added additions we did after importing the grammars.